competition_inputs/BottleJSONReader.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+# G J 2-7-2023
+# Opens and imports JSON File 'BottleJSON.json' as a list
+f = open("competition_inputs/BottleJSON.json")
```

Comment:
Need to add type hints to all of your variables. See https://missourimrr.github.io/docs/python/type-annotations/full-guide.html for a guide